src/Sylius/Behat/Context/Setup/UserContext.php

```diff
@@ -75,6 +75,7 @@ public function __construct(
 
     /**
      * @Given there is user :email identified by :password
+     * @Given there was account of :email with password :password
```

Comment:
The step `@Given there is user :email identified by :password` is already on the master, does it work fine?